lib/sidekiq/cron/job.rb

```diff
@@ -401,18 +400,18 @@ def jid_history_from_redis
       def to_hash
         {
           name: @name,
-          klass: @klass,
+		  klass: @klass.to_s,
```

Comment:
```suggestion
          klass: @klass.to_s,
```
It seems the indentation broke.